auditlog/apps.py

```diff
@@ -11,3 +11,7 @@ def ready(self):
         from auditlog.registry import auditlog
 
         auditlog.register_from_settings()
+
+        from auditlog import models
+
+        models.changes_func = models._changes_func()
```

Comment:
I would move it to `auditlog/models.py`

Reply:
When I had it in the models, it was throwing an error saying that the conf is not found. Do you have any suggestions on how to fix it?